.github/workflows/build-test.yml

```diff
@@ -169,6 +169,7 @@ jobs:
           distribution: 'temurin'
       - name: Build with Maven
         run: |
+          touch gremlin-python/.glv
```

Comment:
why gremlin console depends on gremlin python?

Reply:
This was just the way it was done before since the console tests required python. I'm guessing this was their way of quickly seeing if you had python.